server/utils/risksAndNeeds/offenceAnalysisUtils.ts

```diff
@@ -0,0 +1,128 @@
+import type { OffenceDetail } from '@accredited-programmes/models'
+import type { GovukFrontendSummaryListRowWithKeyAndValue } from '@accredited-programmes/ui'
+
+export default class OffenceAnalysisUtils {
+  static impactAndConsequencesSummaryListRows(
+    offenceDetail: OffenceDetail,
+  ): Array<GovukFrontendSummaryListRowWithKeyAndValue> {
+    return [
+      {
+        key: {
+          text: 'Does the offender recognise the impact and consequences of offending on victim / community / wider society?',
+        },
+        value: {
+          text: this.yesOrNo(offenceDetail.recognisesImpact),
+        },
+      },
+    ]
+  }
+
+  static otherOffendersAndInfluencesSummaryListRows(
+    offenceDetail: OffenceDetail,
+  ): Array<GovukFrontendSummaryListRowWithKeyAndValue> {
+    return [
+      {
+        key: { text: 'Were there other offenders involved?' },
+        value: {
+          text: this.yesOrNo(offenceDetail.numberOfOthersInvolved > 0),
+        },
+      },
+      {
+        key: { text: 'Number of others involved' },
+        value: {
+          text: this.textValue(offenceDetail.numberOfOthersInvolved.toString()),
+        },
+      },
+      {
+        key: { text: 'Was the offender the leader?' },
+        value: {
+          text: this.textValue(offenceDetail.othersInvolvedDetail),
+        },
+      },
+      {
+        key: { text: 'Peer group influences (e.g. offender easily led, gang member)' },
+        value: {
+          text: this.textValue(offenceDetail.peerGroupInfluences),
+        },
+      },
+    ]
+  }
+
+  static responsibilitySummaryListRows(
+    offenceDetail: OffenceDetail,
+  ): Array<GovukFrontendSummaryListRowWithKeyAndValue> {
+    return [
+      {
+        key: { text: 'Does the offender accept responsibility for the current offence(s)?' },
+        value: {
+          text: this.yesOrNo(offenceDetail.acceptsResponsibility),
+        },
+      },
+      {
+        key: {
+          text: 'How much responsibility does s/he acknowledge for the offence(s). Does s/he blame others, minimise the extent of his/her offending?',
+        },
+        value: {
+          text: this.textValue(offenceDetail.acceptsResponsibilityDetail),
+        },
+      },
+    ]
+  }
+
+  static textValue(value: string | null): string {
```

Comment:
This can probably also be `value?: string` to keep it more in line with other possibly nullish method arguments?